libraries/AP_RangeFinder/AP_RangeFinder_PulsedLightLRF.h

```diff
@@ -49,6 +49,7 @@ class AP_RangeFinder_PulsedLightLRF : public AP_RangeFinder_Backend
     uint8_t hw_version;
     uint8_t check_reg_counter;
     bool v2_hardware;
+    bool v3hp_hardware;
```

Comment:
We don't actually need to save this because it's only used inside `init`. Can you move this to be a local variable there?

Reply:
Thanks for pointing out this, I didn't recognize it. But in my new code, I use this variable in `timer` function to jump to PHASE_COLLECT after finishing PHASE_MEASURE. The goal is to update new distance in every call of `timer` function instead of 2 calls(MEASURE-COLLECT), so we can get a new value at 50Hz instead of 25Hz.